include/constants/battle_config.h

```diff
@@ -181,6 +181,7 @@
 #define B_LURE_BALL_MODIFIER        GEN_7 // In Gen7+, Lure Ball's catch multiplier is x5 instead of x3.
 #define B_HEAVY_BALL_MODIFIER       GEN_7 // In Gen7+, Heavy Ball's ranges change. See Cmd_handleballthrow.
 #define B_DREAM_BALL_MODIFIER       GEN_8 // In Gen8, Dream Ball's catch multiplier is x4 when the target is asleep or has the ability Comatose.
+#define B_KINGS_ROCK_BOOST          GEN_7 // In Gen5+, King's Rock is boosted by Serene Grace
```

Comment:
I think I would make this an ability setting for Serene Grace instead since it affects both King's Rock and Razor Fang and it's more of Serene Grace's thing to stack with additional effects in general.

Reply:
updated. ready for review